data/required_tasks.yml

```diff
@@ -4,30 +4,30 @@ pipeline-required-tasks:
   fbc:
     - effective_on: "2025-03-17T00:00:00Z"
       tasks:
-        - [buildah, buildah-10gb, buildah-6gb, buildah-8gb, buildah-remote, buildah-oci-ta, buildah-remote-oci-ta]
+        - [buildah, buildah-remote, buildah-oci-ta, buildah-remote-oci-ta]
```

Comment:
Do we know for sure that no one is using these tasks? If we're not, we could add a new entry to the list of Tasks which exclude it but have an effective date in the future.

Reply:
Do we have any collected data/metrics to see if there were such tasks run recently?

Reply:
The tasks were deprecated half a year ago and expired about two month ago. Conforma will not allow release with such tasks anyway, so I'd say it's safe to delete.